lib/util/miq_apache/miq_apache.rb

```diff
@@ -83,7 +83,7 @@ def self.stop(graceful = true)
 
     def self.httpd_status
       begin
-        res = MiqUtil.runcmd('/etc/init.d/httpd status')
+        res = MiqUtil.runcmd("/etc/init.d/#{MiqApache.service_name} status")
```

Comment:
:+1: I think all `MiqUtil.runcmd` are suspect.

Aah, I thought that was out of scope, but no reason to have line 74 do it one way and this line do it differently.